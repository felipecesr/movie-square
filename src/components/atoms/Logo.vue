<template>
  <img
    class="logo"
    src="@/assets/img/logo.svg"
    alt="MovieSquare! logo">
</template>

<script>
export default {

}
</script>

<style lang="scss" scoped>
.logo {
  width: 15.8rem;
}
</style>
