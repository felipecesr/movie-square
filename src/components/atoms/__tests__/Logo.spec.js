import { shallowMount, RouterLinkStub } from '@vue/test-utils';
import Logo from '../Logo.vue';

describe('Logo.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(Logo, {
      stubs: { RouterLink: RouterLinkStub }
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
