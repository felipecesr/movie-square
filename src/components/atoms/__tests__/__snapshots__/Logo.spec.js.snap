// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo.vue renders correctly 1`] = `
<a>
  <img
    alt="MovieSquare! logo"
    class="logo"
    src="@/assets/img/logo.svg"
  />
</a>
`;
